CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Removed
 
 ### Bug Fixes
+- Changed watches.go to fix log.Info to modify message,and maxWorkers logic ([#2246](https://github.com/operator-framework/operator-sdk/pull/2246/files))
 
```

Comment:
Need to be something that any user who looked that can understand. Maye something as:

> Fix log messages to make clear what are the values used and environment variables used when the operator starts.